src/axom/inlet/Table.hpp

```diff
@@ -941,7 +1118,7 @@ class Table : public Verifiable
   std::vector<AggregateField> m_aggregate_fields;
 
   static const std::string ARRAY_GROUP_NAME;
```

Comment:
Should this be changed to `CONTAINER_GROUP_NAME`? It covers both arrays and dicts right?